README.md

```diff
@@ -61,14 +61,14 @@ Run the ``deploy.sh`` script within a tmux or screen session:
 
 ``` shell
 cd /opt/rpc-openstack
-export RPC_PRODUCT_RELEASE="queens"  # This is optional, if unset the current stable product will be used
+export RPC_PRODUCT_RELEASE="rocky"  # This is optional, if unset the current stable product will be used
 ./scripts/deploy.sh
 ```
 
 #### Configure and deploy the cloud
 
 To configure the installation please refer to the upstream OpenStack-Ansible
-documentation regarding basic [system setup](https://docs.openstack.org/project-deploy-guide/openstack-ansible/queens/configure.html).
+documentation regarding basic [system setup](https://docs.openstack.org/project-deploy-guide/openstack-ansible/master/configure.html).
```

Comment:
'master' here should be 'latest' until rocky releases, when it will become 'rocky'